### ISSUE TITLE
Problem with Card style

### ISSUE THREAD

**Issue by paduvi** [react-native-community/react-native-elements#158]
I'm trying example code:
```
<Card
  title='HELLO WORLD'
  image={require('../images/pic2.jpg')}>
  <Text style={{marginBottom: 10}}>
    The idea with React Native Elements is more about component structure than actual design.
  </Text>
  <Button
    icon={{name: 'code'}}
    backgroundColor='#03A9F4'
    fontFamily='Lato'
    buttonStyle={{borderRadius: 0, marginLeft: 0, marginRight: 0, marginBottom: 0}}
    title='VIEW NOW' />
</Card>
```
It displays error message: 

> View was rendered with explicitly set width/height but with a 0 flexBasis. (This might be fixed by changing flex: to flexGrow:)

Btw, it also shows 

> Unrecognized font family 'Lato'

What can I do to fix this? I'm just newbie to React Native and React Native Element

**Comment by binoy14** [comment]
In terms of font check out this [note](https://github.com/react-native-community/react-native-elements#notes) Lato is not available by default you will have to download it and configure it.

In terms of the View error if you could provide more code it will be more helpful.

**Comment by Monte9** [comment]
@paduvi any updates?

**Comment by DavidKongDesheng** [comment]
@paduvi Got same issue after I set imageStyle={{ flex: 0, height: 300 }} it works, and then remove imageStyle, everything works fine, don't know why, actually nothing changed...

**Comment by binoy14** [comment]
@DavidKongDesheng Take a look at https://github.com/facebook/react-native/issues/11481 let me know it helps

**Comment by vaderdan** [comment]
A quick fix is, at least what worked for me is to add
```
containerStyle={{ flex: 1 }}
wrapperStyle={{flex: 1}}
```
to the card element, but then I noticed that the Card component didn't expand proportionally on the children elements in it. and I had to hardcode it's height

after a bit of digging I found if I modify `flex` style on the Card  (https://github.com/react-native-community/react-native-elements/blob/8774b94046ff135753ab85b618f3fb06842162df/src/containers/Card.js#L41)
It start working normally

Should I send pull request

**Comment by binoy14** [comment]
@vaderdan please do, we will review it. Thank you!

**Comment by vaderdan** [comment]
Hi, I pull requested you - with the #200

**Comment by binoy14** [comment]
Fixed with #200 Thanks you @vaderdan
